MMM-MBTA.js

```diff
@@ -301,21 +301,22 @@ Module.register("MMM-MBTA", {
                 wrapper.appendChild(alertTable);
             } else if (uniqueAlerts.size > 0) {
                 var alertHeader = document.createElement("header");
-                alertHeader.className = "module-header alerts";
+                alertHeader.className = "module-header alerts-header";
                 alertHeader.innerHTML = "Alerts";
                 wrapper.appendChild(alertHeader);
 
-                var alertTable = document.createElement("table");
-                alertTable.className = "small";
+                var alertTable = document.createElement("div");
```

Comment:
Are tables common with MagicMirror modules? Got rid of it as it can cause annoying css behavior.

Reply:
Not sure. It's been so long since I've written this. I'll just take your word for it, and it was probably the primary reason it's been so buggy as well?